src/components/Input/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,14 @@
+import { cx } from '@emotion/css';
+import React from 'react';
+import { TextAreaProps } from './textAreaProps';
+import { textAreaStyle } from './style';
+
+const TextArea = ({ placeholder, text }: TextAreaProps): JSX.Element => {
```

Comment:
please also import the other default textarea props, use {...rest} to forward it.